coolwsd-systemplate-setup

```diff
@@ -140,6 +140,14 @@ fi
 find $LOCALBASE/share -name '*.pcf' | xargs rm -f
 find $LOCALBASE/share -name '*.pcf.gz' | xargs rm -f
 
+# Same for /usr/local/share/fonts
```

Comment:
At the top where LOCALBASE is set there is a conditional where it might be (FreeBSD) using /usr/local already, so this becomes /usr/local/local/share then which doesn't source right. I think we need to skip this new piece if have a LOCALHOST of /usr/local already.

Reply:
Does this deserve the conditional? I mean, would it hurt to have a potentially empty directory?

Reply:
... but please ignore my question, I'm creating a change :-)